cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database")
+        connect = None
+        while not connect:
+            try:
+                connect = connections["default"]
+                connect.cursor()
+            except OperationalError:
+                time.sleep(1)
```

Comment:
The while loop will run indefinitely if the database connection is not established. It would be better to limit the number of attempts or the total waiting time to prevent an infinite loop. Also, you might want to add some delay between the attempts to reduce the load on the database server.